### DISCUSSION TITLE
How to change colours?

### DISCUSSION THREAD

**Discussion by humdingerb** [ohmybash/oh-my-bash#476] [Q&A]
End-user here. Hello. :)

I wonder where I can change colours used in the powerline-multiline theme. Esp. the git branch/status and current working directory on the left side.

I suspect it's defined with [SCM_THEME_PROMPT_CLEAN_COLOR=25](https://github.com/ohmybash/oh-my-bash/blob/master/themes/powerline-multiline/powerline-multiline.theme.sh#L21) and CWD_THEME_PROMPT_COLOR=240
But what is colour "25" and "240"? How to change it to another RGB value?

Thanks a lot!

**Comment by akinomyoga** [comment]
> I suspect it's defined with [SCM_THEME_PROMPT_CLEAN_COLOR=25](https://github.com/ohmybash/oh-my-bash/blob/master/themes/powerline-multiline/powerline-multiline.theme.sh#L21) and CWD_THEME_PROMPT_COLOR=240

Yes.

> But what is colour "25" and "240"?

They are the XTerm's 256 terminal color codes. You can find color tables by searching for e.g. xterm 256 colors. For example, the following page seems to be simple and concise.

- [Xterm 256 colors and examples](https://robotmoon.com/256-colors/)

> How to change it to another RGB value?

The RGB specification is not supported by all terminals.
